docker-compose/web-env-vars.env

```diff
@@ -14,8 +14,8 @@ AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
 AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
 AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
 
-# BFD credentials/settings
+# BFD credentials/settings Change here
 DJANGO_USER_ID_SALT=${DJANGO_USER_ID_SALT}
 DJANGO_USER_ID_ITERATIONS=${DJANGO_USER_ID_ITERATIONS}
-FHIR_URL="https://prod-sbx.bfd.cms.gov"
+FHIR_URL="https://test.bfd.cms.gov"
```

Comment:
Is this an intentional change?

Reply:
I imagine it must be. It just seemed like the kind of things I may accidentally commit after making local testing changes, lol

Reply:
Yea sorry probably not enough context in the description. C4DIC is only enabled in BFD's test env, so this change is meant to point over to that.